Adjust the troubleshooting section to the new version of ApiRule (#1096)
kyma-project/api-gateway: docs/user/troubleshooting-guides/03-00-cannot-connect-to-service/v1beta2/03-01-401-unauthorized-403-forbidden.md

# 401 Unauthorized or 403 Forbidden

## Symptom

When you try to reach your service, you get `401 Unauthorized` or `403 Forbidden` in response.

## Cause 

The error `401 Unauthorized` occurs when you try to access a Service that requires authentication, but you have either not provided appropriate credentials or have not provided any credentials at all. You get the error `403 Forbidden` when you try to access a Service or perform an action for which you lack permission.

## Remedy

> Make sure that you are using a JSON Web Token (JWT) with proper scopes, and it is active.

Reviewer: ```suggestion
Make sure that you are using an active JSON Web Token (JWT) with proper scopes.
```
